charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -21,7 +21,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 55.5.0
+version: 55.5.1
```

Comment:
Shouldn't we bump the minor (not the bugfix) version here?

Reply:
I got conflicting comments on this in the past, but bumped minor now